charts/kubearchive/values.yaml

```diff
@@ -8,33 +8,45 @@ kubearchive:
   # the name of the namespace to deploy kubearchive into
   namespace: "kubearchive"
   # configuration for a test namespace
```

Comment:
You removed the `testNamespace` section, so this comment should be moved to the `watchNamespaces` section where you define the `test` namespace.

Reply:
fixed!